question.html

```diff
@@ -98,63 +98,75 @@ <h1 style="font-family: 'Bubblegum Sans', cursive;"><b>Let's ASK!</b></h1>
-			<small style="margin-top:5px; color:#848484"><em> created: {{todo.timestamp | date:'medium' }} <span ng-show="todo.tags">tags: {{todo.tags}}</span> replies: {{todo.replies}} </em></small>
-
-			<!-- display buttons for question -->
-			<div>
-				<button class="btn btn-default btn-xs glyphicon glyphicon-hand-up" ng-click="doLike(todo)" ng-disabled="$storage[todo.$id]">Like</button>
-				<button class="btn btn-default btn-xs glyphicon glyphicon-hand-down" ng-click="doDislike(todo)" ng-disabled="$storage[todo.$id]">Disike</button>
-				<button ng-show="isAdmin" class="btn btn-default btn-xs glyphicon glyphicon-check" ng-click="toggleCompleted(todo)">Toggle Completed</button>
-				&nbsp;
-				<button ng-show="isAdmin" class="btn btn-default btn-xs glyphicon glyphicon-trash" aria-hidden="true" ng-click="removeTodo(todo)">Del</button>
+	<div class="container">
+	<!-- Question Bounder -->
+	<div class="list-group" style="background-color: #FFFFFF; padding: 10px">
+		<!-- Iterate Questions -->
+		<div class="media" ng-repeat="todo in todos | filter:input | questionFilter:maxQuestion" ng-class="{completed: todo.completed, editing: todo == editedTodo}">
+			<!-- Display Avatar -->
+			<div class="media-left">
+				<a href="#">
+					<img class="media-object" src="http://www.tiptopglobe.com/forum/images/avatars/gallery/Simpsons/simpsons_santas_little_help.gif" alt="...">
+				</a>
 			</div>
-				<hr style="margin-top:5px;margin-bottom:5px">
-
-
-				<!-- iterate replies -->
-				<span class="list-group-item" ng-repeat="reply in todosReplies | filter:{'parentID':todo.$id} : true " style="margin-left:50px; background-color: #FFEDD6" >
-					<!-- display replies -->
-					<h5 class="list-group-item-heading">
-						<span class="badge">{{-reply.order}}</span>
-						<span ng-class="{completed: todo.completed}">{{reply.desc}}</span>
-					</h5>
-					<h5 class="list-group-item-text"></h5>
-					<small style="margin-top:5px; color:#848484">
-						<em>created: {{ reply.timestamp | date:'medium' }}</em>
-					</small>
-					<!-- display buttons for reply -->
-					<div>
-						<button class="btn btn-default btn-xs glyphicon glyphicon-hand-up" ng-click="doLikeReply(reply)" ng-disabled="$storage[reply.$id]">Like</button>
-						<button class="btn btn-default btn-xs glyphicon glyphicon-hand-down" ng-click="doDislikeReply(reply)" ng-disabled="$storage[reply.$id]">Disike</button>
-						&nbsp;
-						<button ng-show="isAdmin" class="btn btn-default btn-xs glyphicon glyphicon-trash" aria-hidden="true" ng-click="removeTodo(todo)">Del</button>
-					</div>
-
-				</span>
+			<!-- Question Body -->
+			<div class="media-body">
+				<!-- Question Heading -->
+				<h4 class="media-heading">
+					<span class="label label-danger" ng-show="todo.new">New</span>
+					<span ng-class="{completed: todo.completed}">{{todo.head}}</span>
+					<span class="label label-primary glyphicon glyphicon-comment">Replies<span class="badge alert-info">{{todo.replies}}</span> </span>
```

Comment:
I suggest placing the Replies Icon at the right side of the like and dislike. 

Like it is right now, it is shifted to the right for long titles:

![bildschirmfoto 2015-11-02 um 22 52 12](https://cloud.githubusercontent.com/assets/1174752/10884723/8d727932-81b4-11e5-8619-daabc4cf0e6e.png)